setup/requirements.txt

```diff
@@ -11,6 +11,7 @@ Pillow<=6.2.1  # Latest Pillow is incompatible with current torchvision, https:/
 pyquaternion>=0.9.5
 scikit-learn
 scipy
+seaborn
```

Comment:
Let's add a comment instead in the visualization script.

Reply:
Changed to plotting using matplotlib instead of seaborn